subiquity/ui/views/filesystem/guided.py

```diff
@@ -319,6 +325,117 @@ def _toggle_guided(self, sender, new_value):
 )
 
 
+class GuidedDiskSelectionViewV2Debug(BaseView):
+    title = "Guided storage configuration for storage version 2 (only for debugging)"
+
+    desc = """
+/!\\ This view is only for debugging. Its goal is to make it easier to reproduce \
+some storage-version=2 bugs that may have occurred using the desktop installer.
+
+Considerations:
+ * The view is incomplete, has no translations, might crash in some scenarios, use \
+it at your own risk...
+ * The allowed / disallowed capabilities are basically ignored. Selecting \
+a scenario will use the DIRECT capability (except for manual partitioning where \
+it will use MANUAL)
+ * For target resize, the recommended size will be used (no slider is implemented).
+ """
+
+    def __init__(
+        self,
+        controller,
+        targets: list[GuidedStorageTarget],
+        disk_by_id: dict[str, Disk],
+    ):
+        self.controller = controller
+
+        buttons = []
+
+        for target in targets:
+            if isinstance(target, GuidedStorageTargetManual):
+                label = "Manual partitioning"
+            elif isinstance(target, GuidedStorageTargetReformat):
+                label = "Install to {} after wiping it".format(target.disk_id)
+            elif isinstance(target, GuidedStorageTargetUseGap):
+                label = "Install to {} in {} gap at offset {}".format(
+                    target.disk_id,
+                    humanize_size(target.gap.size),
+                    target.gap.offset,
+                )
+            elif isinstance(target, GuidedStorageTargetEraseInstall):
```

Comment:
suggest unifying {EraseInstall,Resize} and just fixing the label

Reply:
done, thanks